### ISSUE TITLE
NoMethodError instead of Dentaku::ParseError for specific syntax error

### ISSUE THREAD

**Issue by rhomeister** [rubysolo/dentaku#129]
## Input

`Dentaku::Calculator.new.evaluate!('a a ^&')`

## Expected Behavior

`Dentaku::ParseError: Invalid statement`

## Actual Behavior

```
NoMethodError: undefined method `value' for nil:NilClass
.rvm/gems/ruby-2.3.1/gems/dentaku-2.0.11/lib/dentaku/ast/bitwise.rb:13:in `value'
.rvm/gems/ruby-2.3.1/gems/dentaku-2.0.11/lib/dentaku/calculator.rb:55:in `block in evaluate!'
.rvm/gems/ruby-2.3.1/gems/dentaku-2.0.11/lib/dentaku/calculator.rb:105:in `store'
.rvm/gems/ruby-2.3.1/gems/dentaku-2.0.11/lib/dentaku/calculator.rb:52:in `evaluate!'
```

**Comment by rubysolo** [comment]
Fixed in 2824d562575ee1b9d5a60bd2611c6e8d2c9148d5.  Thanks!

**Comment by rhomeister** [comment]
:+1:
